cves/CVE-2013-2115.yml

```diff
@@ -205,4 +233,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
How can we make sure input is properly sanitized in the future?

Reply:
It was added that there could have been a check for OGNL expressions